src/solver/conv_asm_implicit_gemm_wrw_v4r1_dynamic.cpp

```diff
@@ -280,7 +280,8 @@ size_t ConvAsmImplicitGemmV4R1DynamicWrw::GetWorkspaceSize(const ConvolutionCont
 bool ConvAsmImplicitGemmV4R1DynamicWrw::IsApplicable(const ConvolutionContext& ctx) const
 {
     const auto device_name = ctx.GetStream().GetDeviceName();
-    if(!(StartsWith(device_name, "gfx900") || StartsWith(device_name, "gfx906")))
+    if(!(StartsWith(device_name, "gfx900") || StartsWith(device_name, "gfx906") ||
+         StartsWith(device_name, "gfx908")))
```

Comment:
It seems you allow gfx908 to run v4r1 dynamic? If i remember correct, the kernel for v4r1 dynamic wrw is hard coded as gfx906

Reply:
Yes, we only use v4r1 dynamic for non-xdlops version. I will remove gfx908.

Reply:
removed